pyramid/session.py

```diff
@@ -187,6 +188,9 @@ def BaseCookieSessionFactory(
       Hide the cookie from Javascript by setting the 'HttpOnly' flag of the
       session cookie. Default: ``False``.
 
+    ``samesite``
+      The 'samesite' option of the session cookie. Default ``b'Lax'``.
+
```

Comment:
I think this API should accept a native string... I'm pretty sure the `domain` and `cookie_name` options are not restricted to bytes. My only other comment on this PR right now is that I would err toward bw-compat, meaning cookies do not set a samesite option at all. @bertjwregeer do you have any issues with changing the default to lax?

Reply:
The underlying `Response.set_cookie` API takes bytes, which is why I went with bytes. There are three valid values here: `None`, `b'Lax'`, `b'Strict'`, so it's not a big issue either way.